pkg/kubectl/resource_printer.go

```diff
@@ -503,8 +503,8 @@ var (
 	namespaceColumns                 = []string{"NAME", "STATUS", "AGE"}
 	secretColumns                    = []string{"NAME", "TYPE", "DATA", "AGE"}
 	serviceAccountColumns            = []string{"NAME", "SECRETS", "AGE"}
-	persistentVolumeColumns          = []string{"NAME", "CAPACITY", "ACCESSMODES", "RECLAIMPOLICY", "STATUS", "CLAIM", "REASON", "AGE"}
-	persistentVolumeClaimColumns     = []string{"NAME", "STATUS", "VOLUME", "CAPACITY", "ACCESSMODES", "AGE"}
+	persistentVolumeColumns          = []string{"NAME", "CAPACITY", "ACCESSMODES", "RECLAIMPOLICY", "STATUS", "CLAIM", "STORAGECLASS", "REASON", "AGE"}
+	persistentVolumeClaimColumns     = []string{"NAME", "STATUS", "VOLUME", "CAPACITY", "ACCESSMODES", "REQUESTEDSTORAGECLASS", "AGE"}
```

Comment:
The spacing of the data is off because this column name is so long.  Would it be sufficient to just call it "STORAGECLASS" like above?